app/src/main/java/org/fossasia/badgemagic/ui/EditBadgeActivity.kt

```diff
@@ -51,6 +54,22 @@ class EditBadgeActivity : AppCompatActivity() {
         })
     }
 
+    override fun onCreateOptionsMenu(menu: Menu?): Boolean {
+        menuInflater.inflate(R.menu.open_folder, menu)
+        return super.onCreateOptionsMenu(menu)
+    }
+
+    override fun onOptionsItemSelected(item: MenuItem): Boolean {
+        when (item.itemId) {
+            R.id.open_Folder -> {
+                val intent = Intent(this, DrawerActivity::class.java)
+                intent.putExtra("badge", "badge")
```

Comment:
```suggestion
                intent.putExtra("badge", "badge")
                intent.addFlags(Intent.FLAG_ACTIVITY_CLEAR_TOP)
```

because we want to cancel the last stacked activities.